_inc/lib/class.jetpack-keyring-service-helper.php

```diff
@@ -38,16 +38,35 @@ static function init() {
 		)
 	);
 
+	/**
+	 * Constructor
+	 */
 	private function __construct() {
+		add_action( 'admin_menu', array( __CLASS__, 'add_sharing_menu' ) );
```

Comment:
Should we set a higher priority than normal to be defensive so we don't `add_submenu_page` via this file when it isn't needed?

Reply:
You're right. How does ac9dfc4a06a692051035996e582e3a61b6c8758b look?